msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/GenericAuthority.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.aad.msal4j;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class GenericAuthority extends Authority {
```

Comment:
OidcAuthority ? To match the name of the public API?

Reply:
I named it 'GenericAuthority' to match the style in .NET: https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/main/src/client/Microsoft.Identity.Client/Instance/GenericAuthority.cs

However, I noticed that .NET's authority classes are not public (?), and agree that it's better for the public APIs in one library to match than it is have this bit of SDK consistency. In the latest commit it's been changed to 'OidcAuthority'